_data/retail.yml

```
@@ -503,6 +503,10 @@ websites:
     url: https://www.target.com/
     twitter: Target
     img: target.png
+    tfa:
+      - sms
+      - email
+    doc: https://security.target.com/FAQs
 
```

Let's add a region tag as well while you're at it 😉 
```suggestion
    regions:
      - us

```